index.md

```diff
@@ -5,7 +5,7 @@ title: home
 ### A weekly review club for Bitcoin Core PRs
 
 <span class="question">What is this?</span> &nbsp;A weekly club for reviewing
-Bitcoin Core PRs at **18:00 UTC on Wednesdays** on IRC.
+Bitcoin Core PRs at **{{ site.meeting_time }}** on IRC.
```

Comment:
Hacky but perhaps `Bitcoin Core PRs at **{{ site.meeting_time }}s** on IRC.` to keep "Wednesdays".

Reply:
(or define a `general_meeting_time` and a `meeting_time` in the site settings)

Reply:
I'm ok with losing the s :)

Reply:
I've added the s back in!